app/scripts/modules/ecs/src/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -247,6 +254,7 @@ module.exports = angular
       function buildServerGroupCommandFromExisting(application, serverGroup, mode = 'clone') {
         var preferredZonesLoader = AccountService.getPreferredZonesByAccount('ecs');
 
+        // TODO: not set when called w/ existing template, might need to call buildUpdateServerGroupCommand?
```

Comment:
This appears to be an existing issue, which I didn't believe needed to block the new stuff. 

However, it should be fixed, and would probably boost confidence if it was solved before this went out - I can draft a separate PR to do that (don't anticipate it will be large) so it's addressed.